com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -94,6 +94,8 @@ public class NetworkManager : MonoBehaviour, INetworkUpdateSystem, IProfilableTr
         /// </summary>
         public NetworkSpawnManager SpawnManager { get; private set; }
 
+        internal readonly BufferManager BufferManager = new BufferManager();
```

Comment:
I think this might be better instantiated within the Init method and destroyed within the Shutdown method of NetworkManager?